create/working-with-pipelines-web-console.adoc

```diff
@@ -9,6 +9,11 @@ toc::[]
 You can use the *Administrator* or *Developer* perspective to create and modify `Pipeline`, `PipelineRun`, and `Repository` objects from the *Pipelines* page in the {OCP} web console.
 You can also use the *+Add* page in the *Developer* perspective of the web console to create CI/CD pipelines for your software delivery process.
 
+[IMPORTANT]
+====
+In {OCP} versions 4.12 and 4.13, when using the web console to enter YAML manifests for {openshift-pipelines}, for example when creating pipelines and tasks using the YAML view in the web console, the `tekton.dev/v1` API version is not supported. The `API version in the data does not match the expected API version` error message displays. Use the `apiVersion: tekton.dev/v1beta1` setting instead. {OCP} versions 4.14 and above support the `tekton.dev/v1` API version when using the web console to enter YAML manifests.
```

Comment:
```suggestion
In {OCP} versions 4.12 and 4.13, the `tekton.dev/v1` API version is not supported when using the web console to enter YAML manifests for {openshift-pipelines}. For example, when creating pipelines and tasks using the YAML view in the web console, the `API version in the data does not match the expected API version` error message displays. Use the `apiVersion: tekton.dev/v1beta1` setting instead. {OCP} versions 4.14 and later support the `tekton.dev/v1` API version when using the web console to enter YAML manifests.
```
Suggest going with statement of fact, then cause-and-effect sequence. I had to read the first sentence a few times to parse it. And I think it might be hard to translate, so I would chop it up to smaller bits.